src/include/daos_srv/bio.h

```diff
@@ -966,7 +966,8 @@ struct bio_io_context *bio_mc2ioc(struct bio_meta_context *mc, enum smd_dev_type
  *
  * \return			Zero on success, negative value on error
  */
-int bio_wal_reserve(struct bio_meta_context *mc, uint64_t *tx_id);
+int
+     bio_wal_reserve(struct bio_meta_context *mc, uint64_t *tx_id);
```

Comment:
Is this an unintentional change caused by commit script?

Reply:
yeah, I don't know why clang-format does this.  Probably a bug.  I can try to revert